pkg/minikube/constants/constants.go

```diff
@@ -170,6 +170,11 @@ var LocalkubeCachedImages = []string{
 	// Dashboard
 	"gcr.io/google_containers/kubernetes-dashboard-amd64:v1.6.3",
 
+	// Ingress Controller
```

Comment:
These images get cached on all systems - I don't think we want to add them here.

@r2d4 where do you think we should add these? Maybe in common.sh using the `minikube cache` command in a list?

Reply:
Hi @r2d4 Could you please help to check this?